composer.json

```diff
@@ -35,7 +35,7 @@
         "justinrainbow/json-schema": "^5.0",
         "mi-schi/phpmd-extension": "^4.2",
         "phpmd/phpmd": "^2.4.3",
-        "phpunit/phpunit": "^4.8.35 || ^5.4.3",
```

Comment:
This is because we don't want to update the requirement without updating all the code that uses it. A similar PR (#2754) has already been rejected for the same reason.